### ISSUE TITLE
[Loc]With .NET 7 preview7 installed on locale OS, some strings are unlocalized during installing workload and publish in CLI.

### ISSUE THREAD

**Issue by vdanche** [dotnet/sdk#26493]
.NET 7 preview7 installed on locale OS(e.g DE)

Repro:
Step1:Install wasm-tools

dotnet workload install wasm-tools --skip-manifest-update

            Step3: New blazorWasm 

                            dotnet new blazorwasm 
Step2: dotnet publish with AOT enable
Adding “true” to csproj file to use AOT compilation, like this

<PropertyGroup> <TargetFramework>net7.0</TargetFramework> <RunAOTCompilation>true</RunAOTCompilation></PropertyGroup>

Check if strings are localized well.
Expected Result:
Strings are localized well

Actual Result:
Step1:
Some strings are not localized well during installing workload

![image](https://user-images.githubusercontent.com/65637796/178241075-3ab0c6c4-e793-4a44-83e5-da863def6c11.png)

Step2;
Some strings are not localized well during publishing
E.g(DE)
![image](https://user-images.githubusercontent.com/65637796/178239630-b20ab068-c283-44f2-8e3d-0cc04da36bc0.png)

**Comment by marcpopMSFT** [comment]
I don't believe most localization for .net 7 has been done yet. At least for the SDK, we are enabling that for RC1.

**Comment by vdanche** [comment]
Still repro on NET7 RC2 SDK(7.0.100-rc.2.22471.15)
e.g TK
![image](https://user-images.githubusercontent.com/65637796/192186874-8ea39fec-803c-4edf-a78c-0faa44800b88.png)

**Comment by marcpopMSFT** [comment]
All the strings are fixed in rc1 except for download/done. We'll have to fix those in 7.0.2xx:
https://github.com/dotnet/sdk/blob/release/7.0.1xx-rc1/src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs#L774

@joeloff good workloads introduction bug for @nagilson?

**Comment by vdanche** [comment]
> All the strings are fixed in rc1 except for download/done. We'll have to fix those in 7.0.2xx: https://github.com/dotnet/sdk/blob/release/7.0.1xx-rc1/src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs#L774
> 
> @joeloff good workloads introduction bug for @nagilson?

@marcpopMSFT some strings during publish still unlocalized in NET7 RC2.
![image](https://user-images.githubusercontent.com/65637796/192422787-ba0a8d5f-a5bf-442e-b274-c810687d200c.png)

**Comment by marcpopMSFT** [comment]
@agocke the first three strings listed in the above screenshots come from runtime:
https://github.com/dotnet/runtime/blob/main/src/mono/wasm/build/WasmApp.Native.targets#L323

That last one comes from the Blazor SDK @javiercn 
https://github.com/dotnet/sdk/blob/main/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets#L581

**Comment by vdanche** [comment]
another strings "stripping symbols from..." also should be localized during publish
e.g DE
![image](https://user-images.githubusercontent.com/65637796/198231601-1c517c5b-8784-4c31-9b6d-1918f5d096e9.png)

**Comment by marcpopMSFT** [comment]
I believe that last one is also part of the list from the runtime that @agocke would own.

**Comment by nagilson** [comment]
Unassigning as it's up to those teams to translate their messages
